methods.d.ts

```diff
@@ -29,6 +29,7 @@ import {
   WebhookInfo,
 } from "./types";
 
+import Edited = Update.Edited
```

Comment:
Why are you using `import =` instead of `type =` to define an alias?

Reply:
To preserve JSDoc.

Reply:
Nice. TIL